lib/tasks/forem.rake

```diff
@@ -1,8 +1,10 @@
 namespace :forem do
   desc "Performs basic setup for new Forem instances"
+  # Since we execute this tasks in bin/setup as well as app_initializer:setup
+  # everything happening here needs to be idempotent.
   task setup: :environment do
```

Comment:
I completely forgot about this, it wasn't added by me but afterward by @maestromac. If I had to wager a guess, I'd say we discovered that when running on our systems stack profile fields did not exist which caused problems. This now makes sense because `forem:setup` where we bootstrapped at least a minimal set of fields didn't run there. 

This should be a lot less of an issue once all profile and user setting work is done because nothing in the codebase should be relying on fields that don't exist yet anymore. But we're not quite there and I'm a bit afraid of removing this method just now. I think I'd rather do that as part of https://github.com/forem/forem/pull/12947 at the end.

Reply:
Works for me! That will be a good little bit of tech debt to pay down.